### PR TITLE
test2

### PR DESCRIPTION
**Pull request by deannacodes**
Added text file

### CODE REVIEW

**Comment by ponaroth** [comment]
looks good

**Comment by Doublenyher** [comment]
bad grammer

File: test2.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+awfd
+asfd
```
Comment (GeorgeBeltran) [inline test2.txt]: typo

File: test2.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+asd
+f
```
Comment (Doublenyher) [inline test2.txt]: F

Comment (GeorgeBeltran) [reply test2.txt]: F

File: test2.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+awfd
+asfd
+asd
```
Comment (deannacodes) [inline test2.txt]: ????

**Pull request merged**